src/libraries/System.Linq.Parallel/README.md

```diff
@@ -0,0 +1,16 @@
+# System.Linq.Parallel
```

Comment:
We don't actually own area-System.Linq.Parallel, but unless the @dotnet/area-system-linq-parallel owners object we could add this on their behalf.

Reply:
> We don't actually own area-System.Linq.Parallel, but unless the https://github.com/orgs/dotnet/teams/area-system-linq-parallel owners object we could add this on their behalf.

@eiriktsarpalis We don't object, thank you for your help!